tools/UnitTestApp/TestSetupIndex.js

```diff
@@ -111,9 +111,6 @@ const testSetupRewire = () => {
   // core.setOutput("ModifiedChart", chart.toString());
   const indexFilePath = `${npmRoot}/@fluentui/react-charting/lib-commonjs/index.js`;
   setupIndexFile(indexFilePath, null);
-  // const indexFileContents = require(indexFilePath);
-  // core.setOutput("ModifiedIndex", indexFileContents.toString());
 };
 
-testSetupRewire();
-module.exports=testSetupRewire;
+testSetupRewire();
```

Comment:
why is index.js file in dist folder

Reply:
the index.js file is generated in dist folder by default as a part of ncc build using vercel/ncc

Reply:
then where is its content coming from if the file is autogenerated. And how to change the code in this fle.
I see that you have deleted the script.ts that had the code.